### DISCUSSION TITLE
How can i edit ticket design?

### DISCUSSION THREAD

**Discussion by leonvva** [alfio-event/alf.io#1081] [Q&A]
How can i edit ticket design

When u download pdf ticket. To change that design.

**Comment by cbellone** [comment]
you can do it for a single event:

![image](https://user-images.githubusercontent.com/3385346/167874930-b93e0824-e7bd-4dd3-b124-3913e73cd721.png)

or for an organizer (Configuration -> select organization)

![image](https://user-images.githubusercontent.com/3385346/167875272-c200b6a7-d692-4afe-95c9-3851e3ca2eb8.png)


in both cases, the relevant resource to modify is: **TICKET_PDF**

**Reply by wesleypearson** [reply]
@cbellone is there a way to add images to the ticket PDF and or the Ticket email?
